crates/sui-types/src/base_types.rs

```diff
@@ -87,12 +90,21 @@ pub fn random_object_ref() -> ObjectRef {
     )
 }
 
+/// Type of a Sui object
+#[derive(Clone, Serialize, Deserialize, Ord, PartialOrd, Eq, PartialEq, Debug)]
```

Comment:
we can `derive(JsonSchema)` for this type?

Reply:
Good question--it doesn't work because `StructTag` doesn't derive or implement `JsonSchema`